<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TasksWidget</class>
 <widget class="QWidget" name="TasksWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>198</green>
        <blue>198</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>198</green>
        <blue>198</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="_tasks_type">
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Playing And Waiting</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Playing Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Finished And Canceled</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="_tasks_list"/>
   </item>
  </layout>
  <action name="_cancel_action">
   <property name="text">
    <string>cancel</string>
   </property>
  </action>
  <action name="_repeat_action">
   <property name="text">
    <string>repeat</string>
   </property>
  </action>
  <action name="_finish_action">
   <property name="text">
    <string>finish</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
